views/index.ejs

```diff
@@ -40,6 +42,6 @@
     <script async
     src="https://maps.googleapis.com/maps/api/js?key=<%= apiKey %>&callback=initMap">
     </script>
-    
+    <script src="/components/clock.js"></script>
```

Comment:
@TalesPinto I think it may be a good idea to have the script type as "module" so as to not interfere with the client.js script we've also linked above.